src/arcade/src/Microsoft.DotNet.Helix/Sdk/tools/dotnet-cli/DotNetCli.props

```diff
@@ -3,7 +3,7 @@
 
   <PropertyGroup>
     <IncludeDotNetCli Condition=" '$(IncludeDotNetCli)' != 'true' ">false</IncludeDotNetCli>
-    <AspNetCoreRuntimeVersion>10.0.0-preview.6.25272.112</AspNetCoreRuntimeVersion>
+    <AspNetCoreRuntimeVersion>10.0.0-preview.6.25302.104</AspNetCoreRuntimeVersion>
```

Comment:
I think we could get rid of harcoding this here now that aspnetcore/runtime/sdk versions are always the same